frontend/app/src/components/Sidebar/SidebarNav.test.tsx

```diff
@@ -259,13 +259,6 @@ describe("SidebarNav", () => {
 
     // isActive prop used to style background color, so check that
     expect(links[0]).toHaveStyle("background-color: transparent")
-    expect(links[1]).toHaveStyle("background-color: rgba(151, 166, 195, 0.15)")
-  })
-
-  it("changes the text color when the page is active", () => {
```

Comment:
`expect(screen.getByText("my other page")).toHaveStyle("color: #262730")` now failing due to emotion styled component - removed since covered in e2e testing/snaps